### ISSUE TITLE
🐛 `type T = import;` should be a syntax error

### ISSUE THREAD

**Issue by simon-paris** [biomejs/biome#4982]
### Environment information

```block
CLI:
  Version:                      0.0.0
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           unset
  JS_RUNTIME_NAME:              unset
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    true
  VCS disabled:                 false

Workspace:
  Open Documents:               0
```

### What happened?

This code does not generate any error:
https://biomejs.dev/playground/?code=dAB5AHAAZQAgAFQAIAA9ACAAaQBtAHAAbwByAHQAOwAKAA%3D%3D

This happens on main only.

### Expected result

Import type expressions must have arguments. There should be a syntax error.

cc @fireairforce

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by fireairforce** [comment]
i will fix this~

**Comment by fireairforce** [comment]
i see @togami2864 open a pr for this but close, are you still working on this?

**Comment by togami2864** [comment]
@fireairforce You can go ahead :)
